arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -905,6 +905,44 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/operations": {
```

Comment:
@jhendrixMSFT Thanks will correct this. Also, is it fine, if the examples are added in later PRs (given the special scenario of ours, as we have ~95 distinct operations and of them only ~22 are submitted in this PR)?